life_expectancy/cleaning.py

```diff
@@ -1,21 +1,22 @@
 """cleaning data"""
```

Comment:
The module docstring could be more explicit. For example, explain that this module loads, cleans and saves data related to the life expectancy in europe.